src/service.ts

```diff
@@ -93,12 +93,27 @@ export interface IServiceRequest {
   output: string;
 }
 
+export interface IConfig {
+  serviceUrl: string;
+  rustc: string;
+}
+
 export class Service {
-  static async sendRequest(command: string): Promise<IServiceRequest> {
-    const response = await fetch("//wasmexplorer-service.herokuapp.com/service.php", {
+  static config: IConfig;
+
+  static async sendRequest(body: string, to: string): Promise<IServiceRequest> {
+    if (!this.config) {
+      const configUrl = `${location.protocol}//${location.hostname}${(location.port ? ":" + location.port : "")}/config.json`;
+      this.config = await fetch(configUrl).then(resp => resp.json());
```

Comment:
Extract `IConfig` and `fetch(configUrl).then(resp => resp.json())` into separate .ts module.